### DISCUSSION TITLE
Is there a way to ignore highlights to my own emotes? (Regex)

### DISCUSSION THREAD

**Discussion by hibsbsbs** [Chatterino/chatterino2#3702] [Q&A] [answered]
Let's say I have regex on "hibs" and my own emote has "hibs" in it, is there a way I can make it stop highlighting me? Thanks.

**Comment by LosFarmosCTL** [comment, marked as answer]
If you only want to get mentioned when someone types "hibs" as a separate word I would suggest just turning off regex support for that highlight.

**Reply by hibsbsbs** [reply]
What if it's not an option? I suppose there's a way to ignore certain patterns but my knowledge on regex is almost non existent 🤨

**Reply by LosFarmosCTL** [reply]
Sure, there is, but I’m not sure if I can really help there without understanding the entire issue. Why is it not an option to disable regex, what exactly are you trying to get highlighted?

If you really only want to disable all your emotes not being highlighted while i.e. hibsJKdnfjinsdf still mentioning you I think a negative lookahead would be your best option, something like this: “hibs(?!love|what|laugh|despair)” with your own emote names should work.

**Reply by hibsbsbs** [reply]
It's a little tough to explain, let's just say my friends really like to make puns with my nickname so I enable regex so I get highlighted by them. But that worked pretty well, thanks.

**Reply by hibsbsbs** [reply]
In another note, is there a way to make "hiiiiiibs" or any repetition on the I letter highlight me?

**Reply by LosFarmosCTL** [reply]
Yes there is, just replace hibs with hi+bs

The + means “match the previous character between one and infinite times”, so it would match i.e. on hibs, hiiiiibs or whatever other variation.
